content/manager_webui/plugins-snapshots-page.md

```diff
@@ -70,8 +72,8 @@ Depending on your requirements, the tenant may be prepared according to one of t
 
 ### Deleting a Snapshot
 
-1.  In the Snapshots widget, click the List icon on the far right of the snapshot entry that you want to delete.
-2. Click **Delete** to delete the snapshot from Cloudify Manager.
+1.  In the Snapshots widget, click the trashbin icon on the far right of the snapshot entry that you want to delete.
```

Comment:
trashbin > trash bin
It would be better to use the icon instead of the name, such as:
... click [ ] for the snapshot entry that you want to delete.

Reply:
I don't know how to show the icon here, if it's possible then I agree it's better